charts/kargo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: kargo
 description: Kargo
 type: application
-version: 0.1.0
+version: 0.1.1
```

Comment:
@Tchoupinax this is just a placeholder value. (I wish it could be v0.0.0 or "placeholder" so that would be more obvious, but the linter doesn't allow that.)

The release process will automatically set this correctly.

Reply:
```suggestion
version: 0.1.0
```